### ISSUE TITLE
[Fix]: Logo and address missing on footer

### ISSUE THREAD

**Issue by delight-xxx** [hngprojects/hng_boilerplate_nextjs#1138]
### Description

**Current state**: The logo and address is not visible on the footer
**Resolution:** Logo and address should be on the footer

### Acceptance Criteria

- [ ] The product logo is correctly displayed in the footer on all devices.
- [ ]  The  address is visible and properly formatted in the footer on all devices.
- [ ]  The logo and address are responsive and adjust well to different screen sizes.

### Links

| [`FIGMA LINK`](LINK_HERE)

### Images

![image](URL_HERE)
Design:  <img width="363" alt="Screenshot 2024-08-22 at 11 44 39 AM" src="https://github.com/user-attachments/assets/3272cf5e-3d00-4162-abb7-dfc7c7781477">

Current state: <img width="443" alt="Screenshot 2024-08-22 at 11 23 24 AM" src="https://github.com/user-attachments/assets/b6eb5290-ba35-4bc5-88b9-fc35b79e082c">

**Comment by Muneerat** [comment]
I will work on this

**Comment by Muneerat** [comment]
This is a duplicate to #1208

**Comment by delight-xxx** [comment]
Yes, although this issue was created earlier than 1208. what is the way forward? can you mark the other as duplicate? @incredible-phoenix246
